dist/index.html

```diff
@@ -21,6 +21,14 @@ <h1 class="widgetTitles">Hydration Data</h1>
             <h2 id="dailyWater"></h2>
             <h2>Weekly Water Consumption:</h2>
             <h3 id="weeklyWater"></h3>
+            <form id="form">
+              <label for="date">Date:</label>
+              <input id="dateInput" type="text" name="date" placeholder="yyyy/mm/dd" required>
```

Comment:
I like this placeholder thing. Now to figure out how to make them enter the correct date.